bin/improver-standardise

```diff
@@ -142,6 +171,20 @@ def main():
 
         output_data = output_data.regrid(target_grid, regridder)
 
+        if args.input_landmask_filepath:
```

Comment:
Your bats tests don't cover all the new Errors you are now raising. It might be good to add one for each.

Reply:
Added extra BATS tests.